### DISCUSSION TITLE
[Feature] Person search match mid-string

### DISCUSSION THREAD

**Discussion by ajpanton** [immich-app/immich#11236] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When searching for a person, either in "Explore" or when assigning faces, is there a reason that it matches only the beginning of each part of a name? If I don't remember the exact spelling of a name, but I know how the end of their last name goes, I would like to be able to search for that.

Also, if I've named someone
Firstname "Nickname" Lastname
I'd like to be able to search for the nickname directly instead of prefixing it with the " symbol.

The search could possibly sort the results so that beginning-of-string matches are placed first and mid-string matches last, to maybe make the results a bit more relevant.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5206
